### ISSUE TITLE
Test Result2

### ISSUE THREAD

**Issue by glee1228** [glee1228/netVLad_Sejong#3]
parameter& Dataset :

### **[고정 파라미터]**
Dataset : Sejong University building( in Seoul ) Image Dataset
input image size : ( 128,128,3 )
netVLAD clustering num : 7
training triplet data size : 700 ( 100 * 3 pairs images per class )
optimizer : SGD(learning rate=0.01, decay=1e-6, momentum=0.9, nesterov=True)
triplet loss margin : 1e-4

### **[유동 파라미터]**
base model : VGG16( pretrained weights : 'imagenet')
total epochs : 15
train batch size : 64
T-SNE Result
![vgg16_tsne_15_64](https://user-images.githubusercontent.com/26589942/59327802-9b47ef80-8d25-11e9-9722-b620d60b3d44.png)

base model : VGG16( pretrained weights : 'imagenet')
total epochs : 25
train batch size : 64
T-SNE Result
![vgg16_tsne_25_64](https://user-images.githubusercontent.com/26589942/59327816-aa2ea200-8d25-11e9-8f51-1f82f406a402.png)

base model : xception( pretrained weights : 'imagenet')
total epochs : 15
train batch size : 64
T-SNE Result
![xception_tsne_15_64](https://user-images.githubusercontent.com/26589942/59327846-be729f00-8d25-11e9-9073-7a4232c000c2.png)

base model : xception( pretrained weights : 'imagenet')
total epochs : 15
train batch size : 64
T-SNE Result
![xception_tsne_25_64](https://user-images.githubusercontent.com/26589942/59327856-c5011680-8d25-11e9-9ed1-295090f349c0.png)

base model : inceptionV3( pretrained weights : 'imagenet')
total epochs : 15
train batch size : 64
T-SNE Result
![inception_v3_tsne_15_64](https://user-images.githubusercontent.com/26589942/59327881-dc400400-8d25-11e9-99b6-e016406716da.png)

base model : inceptionV3( pretrained weights : 'imagenet')
total epochs : 20
train batch size : 64
T-SNE Result
![inception_v3_tsne_20_64](https://user-images.githubusercontent.com/26589942/59327887-df3af480-8d25-11e9-88ac-7d60c256c099.png)

base model : densenet121( pretrained weights : 'imagenet')
total epochs : 15
train batch size : 64
T-SNE Result
![densenet121_tsne_15_64](https://user-images.githubusercontent.com/26589942/59327940-042f6780-8d26-11e9-957d-6caa4db31677.png)

base model : densenet169( pretrained weights : 'imagenet')
total epochs : 15
train batch size : 32
T-SNE Result
![densenet169_tsne_15_32](https://user-images.githubusercontent.com/26589942/59327946-072a5800-8d26-11e9-8405-46641f87b16e.png)
